### PR TITLE
Post All_Product_List

### PR DESCRIPTION
**Pull request by AmrutaDhage20**

### CODE REVIEW

File: src/test/java/org/example/PostAllProductList.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    private static final Logger logger = Logger.getLogger("PostAllProductList.class");
+
+    @BeforeTest
+    public void getLoggerDisplay() {
+        PropertyConfigurator.configure("log4j2.properties");
+    }
+
+    @Test(priority = 1)
+    public void validateResponseCode() {
+        Response response = given().when().post(url);
```
Comment (anushreeSonak) [inline src/test/java/org/example/PostAllProductList.java]: Declare response globally

File: src/test/java/org/example/PostAllProductList.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
+package org.example;
```
Comment (anushreeSonak) [inline src/test/java/org/example/PostAllProductList.java]: Package name should be meaningful

**Review by anushreeSonak — COMMENTED** [review]
Please remove unnecessary files

File: src/test/java/org/example/PostAllProductList.java (hunk 1 above)
Comment (AmrutaDhage20) [reply src/test/java/org/example/PostAllProductList.java]: Done

File: src/test/java/org/example/PostAllProductList.java (hunk 2 above)
Comment (AmrutaDhage20) [reply src/test/java/org/example/PostAllProductList.java]: Done

**Review by anushreeSonak — APPROVED** [review]

**Review by amitsahu8 — APPROVED** [review]

**Pull request merged**